terraform/workspace-variables/production_app_env.yml

```diff
@@ -11,3 +11,5 @@ REGISTER_AND_PARTNER_URL: "https://manage-training-for-early-career-teachers.edu
 DOMAIN: support-for-early-career-teachers.education.gov.uk
 GOVUK_WEBSITE_ROOT: support-for-early-career-teachers.education.gov.uk
 GOVUK_APP_DOMAIN: support-for-early-career-teachers.education.gov.uk
+EMAIL_SENDING_ENABLED: false
```

Comment:
Hmmmmmm. This looks like a feature flag. Is it?

If so, I am pretty sure blue-green deploys that we are using will blow away any changes you make to enviroment after an app is deployed. So environment is not the best place for it. 

R&P had a problem like that not too long ago, and they decided to persist feature flags in their database. 

Check out https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/services/feature_flag.rb.

Alternatively, I guess you could activate it in prod by pushing a release changing this value to true. And maybe that is god enough for your needs.

Reply:
Yeah, it's a feature flag. I think the idea is to activate it in prod by pushing a release as you've said. 

We had a look at the feature flag set up in R&P but decided to just go with a quicker option as it's just this one flag. I also had a look at the way it's implemented in [find teacher training](https://github.com/DFE-Digital/find-teacher-training/search?q=feature_flags), unless this creates the same problem you mentioned?

Reply:
I think if you are happy with managing it with prod releases then the current implementation should work just fine.

Reply:
@kicferk1 Yeah this seemed like the simplest way we could have settings we could easily change between different environments (I guess we could also have used the credentials files). I think its fine to have to make a PR to change these values since we only have one "flag" for now 🤷